src/app/_components/Stats/TransactionTypeFilterMenu.tsx

```diff
@@ -0,0 +1,70 @@
+import { useCallback, useMemo } from 'react';
```

Comment:
I recommend co-locating this component with the page that uses it, since it's only being utilized on a specific page.